exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/dml/UpsertTests.kt

```diff
@@ -626,6 +627,34 @@ class UpsertTests : DatabaseTestsBase() {
         }
     }
 
+    @Test
+    fun testUpsertWithUUIDPrimaryKey() {
+        val tester = object : UUIDTable("upsert_test", "id") {
+            val key = integer("test_key").uniqueIndex()
+            val value = text("test_value")
+        }
+
+        // At present, only Postgres returns the correct UUID directly from the result set.
+        // For other databases incorrect ID is returned from the 'upsert' command.
+        withTables(excludeSettings = TestDB.entries - listOf(TestDB.POSTGRESQL, TestDB.POSTGRESQLNG), tester) { testDb ->
```

Comment:
For the future, there are some available sets for common `TestDB` collections in its companion object, if you ever need them.